grakn-test/src/main/java/ai/grakn/test/GraknTestEnv.java

```diff
@@ -68,6 +74,19 @@ public static void ensureCassandraRunning() throws Exception {
     }
 
     /**
+     * Create a configuration for use in tests, using random ports.
+     */
+    static GraknEngineConfig createTestConfig() {
+        GraknEngineConfig config = GraknEngineConfig.create();
+
+        Integer serverPort = getEphemeralPort();
```

Comment:
What would happen if the port generated here conflicted with one of our in-use ports?